src/writer/fail_on_skipped.rs

```diff
@@ -57,7 +57,7 @@ where
         Option<&gherkin::Rule>,
         &gherkin::Scenario,
     ) -> bool,
-    Wr: for<'val> writer::Arbitrary<'val, W, String>,
+    Wr: Writer<W>,
```

Comment:
@ilslv could you give more explanation behind this change?

Reply:
@tyranron this was just a redundant bound, that for some reason was there. I bumped into it, because `writer::Or` isn't implementing `writer::Arbitrary`, so `.fail_on_skipped()` didn't work with `writer::Or<writer::Basic, writer::Libtest>`